### DISCUSSION TITLE
When I run 'webui --upgrade', I get the following error. How can I resolve it? thx

### DISCUSSION THREAD

**Discussion by sevenlucas7** [vladmandic/sdnext#1049] [Q&A]
Apple  M2 macbook
<img width="610" alt="iShot_2023-05-18_15 09 48" src="https://github.com/vladmandic/automatic/assets/131904649/e5458809-e34a-462f-a76f-ff941cd61f34">

**Comment by vladmandic** [comment]
Log is cropped just before actual error, no way of knowing what comes next.
